runtime/j9vm/jvm.c

```diff
@@ -4967,19 +4967,26 @@ JVM_Accept(jint descriptor, struct sockaddr* address, int* length)
 
 			returnVal = select(descriptor+1, &fdset, 0, 0, &tval);
 		} while(returnVal == 0);
-		retVal = accept(descriptor, address, &socklen);
+
+		do {
+			retVal = accept(descriptor, address, &socklen);
+		} while (errno == EINTR);
 		*length = (int)socklen;
 	}
 #elif defined (WIN32)
 	{
-		SOCKET socketResult = accept(descriptor, address, length);
+		do {
+			SOCKET socketResult = accept(descriptor, address, length);
+		} while (errno == EINTR);
 		retVal = (jint)socketResult;
 		Assert_SC_true(socketResult == (SOCKET)retVal);
 	}
 #else
 	{
 		socklen_t socklen = (socklen_t)*length;
-		retVal = accept(descriptor, address, &socklen);
+		do {
+			retVal = accept(descriptor, address, &socklen);
+		} while (errno == EINTR);
```

Comment:
Linux and AIX documentation for `accept` are similar.

Better check: `} while ((-1 == retVal) && (EINTR == errno));`

Linux `accept` documentation: http://man7.org/linux/man-pages/man2/accept.2.html